packages/jest-resolve/src/index.js

```diff
@@ -14,9 +14,9 @@ import type {ErrorWithCode} from 'types/Errors';
 
 import fs from 'fs';
 import path from 'path';
-import nodeModulesPaths from './node_modules_paths';
-import isBuiltinModule from './is_builtin_module';
-import defaultResolver from './default_resolver.js';
+import nodeModulesPaths from './nodeModulesPaths';
+import isBuiltinModule from './isBuiltinModule';
+import defaultResolver from './defaultResolver.js';
```

Comment:
we can remove the `.js` extension :)

Reply:
@thymikee Not able to understand why test failed

Reply:
We have an e2e suite where `default_resolver` is still used:
https://github.com/facebook/jest/blob/a91ea0ea2b62ecd272382de1496ade636160d7c8/e2e/custom-resolver/resolver.js#L3-L5

Change the file path and it should be good to go